src/components/navigation/document-tab-panel.tsx

```diff
@@ -141,13 +141,13 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
   }
 
   private handleDocumentSelect = (document: DocumentModelType, networkClassHash?: string,
-    networkClassName?: string, networkUserName?: string, networkUserId?: string) => {
+    networkUsername?: string) => {
```

Comment:
Maybe this parameter should be named `networkUserId` and then it becomes a `networkUsername` when the `@portal` is appended?